logging/LOGGING_RULES.md

```diff
@@ -0,0 +1,40 @@
+It's a draft of the logging rules for the project.
```

Comment:
remove this file. add it as a comment/attachment to https://bloxxx.atlassian.net/browse/IO1-2512